sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 5.1.0-beta.2 (Unreleased)
+## 5.1.0-beta.2 (2020-10-02)
```

Comment:
It would be better to put the release date when we know for sure when we are going to release. 
Also, code complete is Friday and release prob Tuesday next week.